.docker/Dockerfile.devenv

```diff
@@ -0,0 +1,3 @@
+FROM docker/dev-environments-ruby:stable-1
+RUN gem install bundler jekyll
+CMD ["bundle", "install"]
```

Comment:
Wondering, won't we miss all the env-vars (and potentially custom things) that are in the main dockerfile? Would it be possible to "compose" a dev-environment friendly environment in which we copy the tools from `<some image>` for the integration? https://github.com/docker/docker.github.io/blob/77a86298a373d06802cc8096dca127c4dceb38df/Dockerfile#L14-L41

Reply:
Wondering if "dev environment" in this case should be "ruby + jekyll" or a dev-environment with "docker installed" (as the build process of this project is a docker container (which _happens_ to be using `jekyll` as one of it's components).

Reply:
Not sure to understand 🤔 , we just added a Dockerfile for the dev flow. 
This one is used for production, no?

Reply:
Depending on the build-args, the Dockerfile is for "production" or for local development/preview.

This repository (for the "development flow") has no dependencies, other than docker, because steps to build the documentation are defined in the Dockerfile, which can be built (and run) with `docker-compose up --build`.

In the Dockerfile, this includes Jekyll (with github-pages) (for some steps) as well as bash/curl for other steps, and runs an nginx server to view the results.

I think the discrepancy here with the "dev environment" container is that that container sets up a Ruby/Jekyll environment which will only provide "some" of those. From that perspective, a better reflection of the "host" environment (inside a dev-container) would be a dev-container containing the `docker` (and `docker compose`) CLI (with `docker-compose`).